src/Tooltip/Tooltip.js

```diff
@@ -314,7 +319,7 @@ module.exports = kind(
 	* @private
 	*/
 	positionChanged:function () {
-		Popup.prototype.positionChanged.apply(this, arguments);
+		Popup.prototype.positionChanged && Popup.prototype.positionChanged.apply(this, arguments);
```

Comment:
Surprised this never brought any problems before. There's no `positionChanged` in `enyo/Popup`. We should get rid of this line instead.